### ISSUE TITLE
Send request shortcut doesn't work after I switch tabs

### ISSUE THREAD

**Issue by alexandruirimiea** [a85/POSTMan-Chrome-Extension#3161]
## App Details:

```
Postman for Windows
Version 5.0.0
win32 10.0.15063 / x64
```

## Issue Report:
 1. Did you encounter this recently, or has this bug always been there: I never tested this before
 2. Expected behaviour: the shortcut used for sending request should work when switching to another tab

Steps to reproduce the problem:
1. Send request using `Ctrl + Enter`
2. Switch to another tab using `Ctrl + Tab`
3. Send request using `Ctrl + Enter`

**Comment by sdnts** [comment]
@alexandruirimiea I was unable to reproduce this. I was able to switch tabs using `Cmd+1 / 2 / 3` and send the request using `Ctrl + Enter`. 
When you say you switch to another tab, do you mean you're doing `Ctrl + 1 / 2/ 3` etc. right?

**Comment by alexandruirimiea** [comment]
@madebysid No, I was using `Ctrl + Tab`.

**Comment by sdnts** [comment]
Oh okay. Even with `Ctrl + Enter`, I'm able to send requests with `Ctrl + Enter` after switching tabs. Can you open up your DevTools (View > Show DevTools) and see if there are any errors there?
Also, do you have multiple Postman Windows open?

**Comment by alexandruirimiea** [comment]
Both _DevTools_ and _Postman Console_ don't show anything when problem reproduces. When I switch to another tab using `Ctrl + Tab`, it's like the window loses focus , but it doesn't (because the interface still responds to other commands). 

Are you using the same version as I use? Btw I work on _Windows 10 Pro x64_, version _1703_, build _15063.296_

**Comment by markacianfrani** [comment]
I'm actually have this same issue on macOS 10.12.5 Postman 5.0.0. I've always had this issue. It's exactly like @alexandruirimiea described in that its like the window loses focus. I'm constantly having to manually click the Send button after cmd + tabbing. I have a hard reproducing it sometimes because if I click send, tab out, and then tab back in immediately, it sometimes works.

**Comment by jayceekay** [comment]
also having this issue on mac OS sierra 10.12.5

**Comment by sivcan** [comment]
I suggest you all to update to the [latest version](https://www.getpostman.com/apps) of the app.
Switching done with Cmd and number keys is working fine.

**Comment by steresi** [comment]
I have this very same problem on Mac OS, running the latest version of Postman. Command-Return does not always activate the Send button. I can reproduce the problem because it happens so often.

To solve the problem, I have to hit tab, try again, hit tab, try again, etc... until I have tabbed into whatever field Command-Return works in (there is no indication which field is in focus)!

**Comment by fmiqbal** [comment]
Same problem with me,
my Env :

Postman for Linux
Version 6.4.2
linux 4.18.12-arch1-1-ARCH / x64

- Ctrl tab seems to always work to switch tab, but ctrl 1 /2 /3/ 4 after ctrl tab doesnt seems to work
- Ctrl + Number works once, then it lost focus to nowhere and cant ctrl + number again (except ctrl + tabs, it always work)
- it happen in any UI option, with/without sidebar, two pane/single pane
- Reinstall the package (in case of mine using trizen from aur https://aur.archlinux.org/packages/postman-bin/) also did not solve

EDIT:
problem still occur in 6.4.4 and i found some new beahviour
- Ctrl + number will always work if you click the tab first
- But once the focus is changed to other element inside the window (inside the tab) I only can use the Ctrl + number once and it doesnt work again

Console only show:

![image](https://user-images.githubusercontent.com/8409391/46915719-7efa8400-cfd9-11e8-9c48-3ad36910568a.png)

**Comment by DaviGSilva** [comment]
If you click in the tab and after use ctrl+tab, the ctrl+enter works. Its a very annoying bug!

**Comment by rchurianov** [comment]
**Describe the bug**
Can not execute request with "Ctrl + Enter" after switching to a new tab (request) with "Ctrl + Tab" in Windows native app.

**To Reproduce**
Steps to reproduce the behavior:
1. Have at least two requests in Windows native app
2. Change to a different tab with "Ctrl + Tab"
3. Hit "Ctrl + Enter"
4. Nothing happens

**Expected behavior**
After I switch to a new tab with Ctrl + Tab inside native app I expect request from the current tab to be executed after hitting Ctrl + Enter

**App information (please complete the following information):**
Native App
v6.5.2
Windows 10 1709 16299.716

**Additional context**
Basically all keyboard shortcuts do not respond after switching to anew tab with Ctrl + Tab. No reaction after hitting Ctrl + L | Ctrl + ? etc. #

**Comment by fmobus** [comment]
Same problem for me on MacOs native app using `cmd+shift+]`. This forces me to use the mouse.

Version 7.1.1 (7.1.1)

**Comment by nagy135** [comment]
This is far from **closed** ....issues like this probably never gets fixed. What I did is develop my own "clicker script" with predefined locations and sequences of clicking. So I basically press a bind and it moves my cursor to click on that stupid button. So sad to be forced to do this but hey, at least [this](https://raw.githubusercontent.com/nagy135/scripts/master/clicker) wont break and [xdotool](https://github.com/jordansissel/xdotool) is all you need
